code/game/verbs/ooc.dm

```diff
@@ -143,6 +143,7 @@
 	log_ooc("(LOCAL) [mob.name]/[key] (@[mob.x],[mob.y],[mob.z]): [msg]")
 	var/list/heard
 	var/mob/living/silicon/ai/AI
+	var/mob/name = src.mob.name
```

Comment:
Absolutely no reason to make this a `/mob/`
In fact, this var has absolutely no reason to exist, remove it.

Reply:
It looks a bit cleaner with the var tbh

Reply:
I don't want to paste src.mob.name three times that's why I'm reluctant to remove the var

Reply:
>It looks a bit cleaner with the var tbh

What the fuck am I reading

Reply:
Oh but I already removed the /mob, I believe src.mob.name is needed because this is a client verb, so having it check for the mob's name the client is inhabiting is good
The variable is parsed 3 times, once for normal client, once for what admin hears, and once for what AI hears